file-attachments.mdx

```diff
@@ -0,0 +1,184 @@
+---
+ title: 'File Attachments: Databases can now store files and images'
+ description: 'Extending a database record to include a new file column type with a global CDN, common security boundaries and image transformations.'
+ image:
+   src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/fa_blog_illustration.jpeg
+   alt: 'File Attachments: Databases can now store files and images'
+ author: Sorin Toma
+ date: 08-30-2023
+ published: true
+ slug: file-attachments
+ ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/fa_blog_illustration.jpeg
+---
+
+Today, as part of [our launch week](https://xata.io/blog/launch-week-august-2023), we’re beyond excited to announce a feature that we've wanted to add ever since we started Xata: File Attachments. Think of it as having a new database column type where you can store files of any size, and behind the scenes they are stored in AWS S3 and cached through a global CDN. With this approach, files simply become part of a database record. For example, they respect the same security boundary -- if you delete a record, the files also get deleted. Image file types also get some extra functionality allowing you to request them at any size and style with built-in transformations.
```

Comment:
Changing the example because didn't match too well the "security boundary' part.

```suggestion
Today, as part of [our launch week](https://xata.io/blog/launch-week-august-2023), we’re beyond excited to announce a feature that we've wanted to add ever since we started Xata: File Attachments. Think of it as having a new database column type where you can store files of any size, and behind the scenes they are stored in AWS S3 and cached through a global CDN. With this approach, files simply become part of a database record. For example, they respect the same security boundary -- if you can access a record, you can also access its attached files. Image file types also get some extra functionality allowing you to request them at any size and style with built-in transformations.
```